app/generators/questionnaire_generator.rb

```diff
@@ -142,6 +144,15 @@ def generate_radio(question)
                 ])
     end
 
+    def generate_time(question)
+      safe_join([content_tag(:p, question[:title].html_safe, class: 'flow-text'), time_body(question)])
+    end
+
+    def time_body(question)
+      tag_options = { type: 'time', id: question[:id], required: true, class: 'validate' }
+      tag(:input, tag_options)
+    end
+
```

Comment:
Zou je dit tijdselement een kleinere breedte kunnen geven? bijv s2 of s1 ipv s6?

Reply:
Zou je een stepsize van een kwartier kunnen opgeven? Verder is het prima.